pom.xml

```diff
@@ -979,7 +979,7 @@
 
         <plugin>
           <artifactId>maven-compiler-plugin</artifactId>
-          <version>3.5.1</version>
```

Comment:
No, special need, the goal is to keep up to date the maven plugins, in particular those who can have performance improvements for the build.